.env

```diff
@@ -1 +1,2 @@
 REACT_APP_API=http://localhost:8080
+REACT_APP_GITHUB_CLIENT_ID=eafec0dcd096e688249d
```

Comment:
That is the default for localhost, i'll need to create another Client_ID and secret for dev under the new sandbox.maryvilledevcenter.io